src/api/resourceApi.ts

```diff
@@ -248,6 +248,7 @@ export const getResourceContents = async (identifier: string): Promise<ResourceC
     method: 'GET',
   });
   const resourceContent: ResourceContents = emptyResourceContent;
+  resourceContent.additionalContent = [];
```

Comment:
skjønner ikke hvorfor resourceContent ikke tømmes ordentlig. emptyResourceContent har jo tom additionalContent ???
Fikk problemer med at additionalContent bare økte for hver gang man brukte back-knappen

Reply:
Prøvde å fremprovosere buggen, men klarte ikke. Kan du gi en demo?

Reply:
kan ta det på kontoret ?